packages/relayer/processor/process_message.go

```diff
@@ -461,3 +457,47 @@ func (p *Processor) saveMessageStatusChangedEvent(
 
 	return nil
 }
+
+func (p *Processor) setGasTipOrPrice(ctx context.Context, auth *bind.TransactOpts) error {
+	gasTipCap, err := p.destEthClient.SuggestGasTipCap(ctx)
+	if err != nil {
+		if IsMaxPriorityFeePerGasNotFoundError(err) {
+			auth.GasTipCap = FallbackGasTipCap
+		} else {
+			gasPrice, err := p.destEthClient.SuggestGasPrice(context.Background())
+			if err != nil {
+				return errors.Wrap(err, "p.destBridge.SuggestGasPrice")
+			}
+			auth.GasPrice = gasPrice
+
+			return nil
+		}
+	} else {
+		auth.GasTipCap = gasTipCap
+
+		return nil
```

Comment:
dont think this return nil is needed, since it just returns nil anyuways below this statement.